### DISCUSSION TITLE
Is there a way to see request/response history in the desktop app?

### DISCUSSION THREAD

**Discussion by jase52476** [hoppscotch/hoppscotch#4110] [Q&A] [answered]
I can't figure out how to see my previous requests/responses or see the logs.

**Comment by Cadogantes** [comment]
Same question - are responses being cached somwhere? Can we see responses for previously executed requests?

**Reply by liyasthomas** [reply, marked as answer]
> Hoppscotch helps you to keep track of your requests and responses. You can access the history of your requests by clicking on the ”History” icon on the side panel.

Documentation: https://docs.hoppscotch.io/documentation/features/history

**Reply by edwjusti** [reply]
I can see the request history, however when I click on the entries it doesn't show the response. Is there any way to see this in Hoppscotch?

**Reply by liyasthomas** [reply]
I'm afraid, as of now, Hoppscotch only save request entries to History and will omit responses. We're evaluating multiple way to implement this feature and discussions are ongoing in #1875.
